src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts

```diff
@@ -482,4 +487,10 @@ export class DataChartComponent implements OnInit, OnDestroy {
   getChartData(){
     return [{ id:this.chartConfig.id ,data: this.chartData , selectedFilters:this.currentFilters }];
   }
+  openDialog() {
```

Comment:
Every call to the method will create a new instance of the dialog. Make sure to close the previously opened ones to avoid memory leaks.

Reply:
resloved